src/NovelRT/Maths/GeoBounds.cpp

```diff
@@ -12,11 +12,9 @@ namespace NovelRT::Maths {
     auto corner0 = getCornerInWorldSpace(0);
     auto corner2 = getCornerInWorldSpace(2);
 
-    if (point.getX() >= corner0.getX() && point.getX() <= corner2.getX() && point.getY() <= corner2.getY()
-      && point.getY() >= corner0.getY())
-      return true;
+      return point.getX() >= corner0.getX() && point.getX() <= corner2.getX() && point.getY() <= corner2.getY()
+             && point.getY() >= corner0.getY();
```

Comment:
I assume you ran the unit tests after changing this? Or are there no unit tests for `GeoBounds` yet? (I've been busy getting the tests caught up, sorry >_<).

Reply:
@RubyNova no tests for `GeoBounds` at the moment

Reply:
OK, then we really ought to add some.

We can't have changes like this without the tests to make it airtight, really.

Maybe @Laiot would like to add some?